stickynav.html

```diff
@@ -58,7 +58,6 @@ <h2>Content One</h2>
         quae excepturi facere, iure rerum sequi? Illum velit delectus dicta et
         iste dolorum obcaecati minus odio eligendi!
       </p>
-
       <h2>Content Two</h2>
       <p class="container-content2">
         Lorem ipsum dolor sit amet consectetur, adipisicing elit. Pariatur
```

Comment:
Dummy review 2 - This review has multiline selected. Respond to replying to this comment and also resolve the conversation.